README.md

```diff
@@ -46,6 +46,19 @@ Example:
 $ node cli.js reassign-repository-roles -o OneGHEOrg --old-role "write" --new-role "maintain"
 ```
 
+### Reassign Repository Roles
+Given a GitHub Organization, find each team repository with a specific role, and update this role to a new one.
```

Comment:
```suggestion
Given a GitHub Organization, find each team member with a specific role, and update this role to a new one.
```

Reply:
Not sure I'm right with the suggestion, but seems like this line in the readme needs clarification/correction.